packages/embark/src/lib/core/config.js

```diff
@@ -95,7 +96,17 @@ Config.prototype.loadConfigFiles = function(options) {
   this.embarkConfig = fs.readJSONSync(options.embarkConfig);
   this.embarkConfig.plugins = this.embarkConfig.plugins || {};
 
-  this.plugins = new Plugins({plugins: this.embarkConfig.plugins, logger: this.logger, interceptLogs: interceptLogs, events: this.events, config: this, context: this.context, env: this.env, version: this.version});
+  this.plugins = new Plugins({
+    plugins: this.embarkConfig.plugins,
+    logger: this.logger,
+    interceptLogs: interceptLogs,
+    events: this.events,
+    config: this,
+    context: this.context,
+    env: this.env,
+    version: this.version,
+    embarkPkg: this.pkg
```

Comment:
Added `embarkPkg` here.

Reply:
We could, instead of putting it in the plugin definition (`embark.embarkPkg`), it could be only in `embark.config.package`

Reply:
Don't need `embarkPkg` here anymore

Reply:
This is an outdated change :)